contracts/protocol/PositionManager.sol

```diff
@@ -335,16 +335,13 @@ contract PositionManager is
     function needClosePositionBeforeOpeningLimitOrder(
         uint8 _side,
         uint256 _pip,
-        uint128 _quantity,
```

Comment:
unused param. can compare _pQuantity and _quantity in module limit order instead of passing param into manager to save gas